src/ResourceManager/DataMigration/ChangeLog.md

```diff
@@ -18,4 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
-* Set minimum dependency of module to PowerShell 5.0
```

Comment:
@hitenjava please bring back the line that was deleted

Reply:
Done